### PR TITLE
Do not abort when metadata log is missing

### PR DESCRIPTION
**Pull request by linsword13**

### CODE REVIEW

File: var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py [hunk 1]
```diff
@@ -123,16 +125,20 @@ def _process_id_list(self):
                 f.write(", ".join(sorted(ids)))
 
     def _process_physical_hosts(self):
+        run_dir = self.expander.expand_var("{experiment_run_dir}")
+        log_path = os.path.join(
```
Comment (douglasjacobsen) [inline var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py]: Should we use the `get_file_path` util here?

File: var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py [hunk 2]
```diff
@@ -145,9 +151,7 @@ def _process_physical_hosts(self):
                     level2_groups.add(levels[2])
 
         with open(
-            self.expander.expand_var(
-                "{experiment_run_dir}/gcp-metadata.topology_summary.log"
-            ),
+            os.path.join(run_dir, "gcp-metadata.topology_summary.log"),
```
Comment (douglasjacobsen) [inline var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py:157]: Same comment here about the `get_file_path` util.

File: var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py (hunk 1 above)
Comment (linsword13) [reply var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py]: Good call, updated.

File: var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py (hunk 2 above)
Comment (linsword13) [reply var/ramble/repos/builtin/modifiers/gcp-metadata/modifier.py:157]: Here I think it's fine to not use it, as it opens this file with `w+` mode (and only write to it.)

**Review by douglasjacobsen — APPROVED** [review]

**Pull request merged**